model.py

```diff
@@ -335,11 +335,11 @@ class Separator(nn.Module):
          post-processing stage. Zeroed if only one target is estimated.
          defaults to 1.
 
-    residual: str or None
-        adds an additional residual target with provided name, obtained by
+    residual: bool
+        adds an additional residual target, obtained by
```

Comment:
residual is now just a boolean. Anyways a `str` did not make sense since we now output a Tensor